tests/helpers/assertDatabaseHas.ts

```diff
@@ -1,14 +1,17 @@
 import { sql } from 'slonik';
 import { getPool } from '../../database';
-import {boolean} from "zod";
+import { z } from "zod";
 
 async function assertDatabaseHas(table: string, conditions: Record<string, any>) {
     const pool = await getPool();
     const conditionFragments = Object.keys(conditions).map(key =>
         sql.fragment`${sql.identifier([key])} = ${conditions[key]}`
     );
     const whereCondition = sql.join(conditionFragments, sql.fragment` AND `);
-    const query = sql.type(boolean())`
+    const existsSchema = z.object({
```

Comment:
question: do you like this happening here or does this just feel like a kludge to make the validation hook happy? If it's the latter I don't see a reason why this one couldn't just be a sql.unsafe`` query which I assume doesn't get validated. Put another way, do you think it's helpful here to asser the return type of this query?

Reply:
I don't think it's helpful, and I have no issue using sql.unsafe here. I had it that way when I first wrote the helper, but the slonik docs said not to leave it in production code, so I took it out.

Reply:
yeah I get why they'd say that I suppose. If you're using slonik over just `postgres` it's likely because you want type safety armor and no point in shooting random buried holes in it I suppose.